packages/web-components/src/styles/states/index.ts

```diff
@@ -0,0 +1,61 @@
+import { css } from '@microsoft/fast-element';
+
+/**
+ * Selector for the `primary` state.
+ * @public
+ */
+export const primaryState = css.partial`:is([state--primary], :state(primary))`;
```

Comment:
Nice! I like these shared selectors. To help enforce naming consistency, we might also want to create some common type/interface where possible. For instance, we can have a common `Size` type and then `ButtonSize` can pick a set of properties that it needs. That way, the strings will be defined only in the common types.

Reply:
Good suggestion for follow-up!